shared/router-v2/header/index.desktop.tsx

```diff
@@ -180,7 +180,7 @@ class Header extends React.PureComponent<Props> {
               />
             )}
             <Kb.Box2 direction="horizontal" style={styles.topRightContainer}>
-              {flags.kbfsOfflineMode && <SyncingFolders />}
+              {flags.kbfsOfflineMode && <SyncingFolders negative={true} />}
```

Comment:
instead of making this always negative, can you make it negative depending on the color of the background?

Reply:
Do we even know that?